tests/migration_test.go

```diff
@@ -161,24 +158,6 @@ var _ = Describe("[Serial][rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][leve
 
 		tests.SkipIfMigrationIsNotPossible()
 
-		// Taints defined by k8s are special and can't be applied manually.
-		// Temporarily configure KubeVirt to use something else for the duration of these tests.
-		if tests.IsUsingBuiltinNodeDrainKey() {
-			drain := "kubevirt.io/drain"
-			cfg := defaultKVConfig()
-			cfg.MigrationConfiguration.NodeDrainTaintKey = &drain
-			tests.UpdateKubeVirtConfigValueAndWait(cfg)
-		}
-	})
-
-	AfterEach(func() {
-		currKubeVirt := util.GetCurrentKv(virtClient)
-
-		// if revision changed, patch data and reload everything
-		if currKubeVirt.ResourceVersion != originalKubeVirtConfig.ResourceVersion {
-			originalKubeVirtConfig = tests.UpdateKubeVirtConfigValueAndWait(originalKubeVirtConfig.Spec.Configuration)
-		}
```

Comment:
what restore teh kubevirt object now if we remove this "AfterEach"?

Reply:
That is all done now like in other tests in the beforetestcleanup.

Reply:
ah right, i was getting this confused with the logic that restores the KV config  to whatever it was before the test suite runs... but that's different.